client/packages/purchasing/src/purchase_order/api/operations.graphql

```diff
@@ -347,3 +347,24 @@ mutation updatePurchaseOrderLine(
     }
   }
 }
+
+mutation deletePurchaseOrderLines($ids: [String!]!, $storeId: String!) {
+  deletePurchaseOrderLines(ids: $ids, storeId: $storeId) {
+    id
+    response {
+      ... on DeleteResponse {
+        id
+      }
+      ... on DeletePurchaseOrderLineError {
+        __typename
+        error {
+          description
+          ... on RecordNotFound {
+            __typename
+            description
```

Comment:
I wonder if not being able to edit should also come through as a structured error? 

But probably doesn't matter